checkov/openapi/checks/resource/v2/Oauth2SecurityRequirement.py

```diff
@@ -16,7 +16,7 @@ def __init__(self) -> None:
         super().__init__(name=name, id=id, categories=categories, supported_entities=supported_resources,
                          block_type=BlockType.DOCUMENT)
 
-    def scan_entity_conf(self, conf: dict[str, Any], entity_type: str) -> tuple[CheckResult, dict[str, Any]]:  # type:ignore[override] # return type is different than the base class
+    def scan_openapi_conf(self, conf: Dict[str, Any], entity_type: str) -> Tuple[CheckResult, dict[str, Any]]:
```

Comment:
same here for the types, just use them as before 🙂 same goes for the other checks you adjusted